FlowCal/excel_ui.py

```diff
@@ -403,6 +404,9 @@ def process_beads_table(beads_table,
                      np.percentile(beads_sample_gated[:, sc_channels[1]],
                                    95) * (10**0.75),
                      )
+                # Beads have a tight distribution, so less smoothing should be
+                # applied for visualization
+                density_params['sigma'] = 5.
```

Comment:
this is specific to beads? I was originally gonna say that the default on `plot.density2d()` should just be updated, but you think the current default (`sigma=10`?) is still appropriate for cells?